app/popup.html

```diff
@@ -28,9 +28,14 @@
     <!-- Build Item -->
     <script id='build_item' type='text/x-handlebars-template'>
       <div class='branch_info'>
-        <a target='codeship' href='http://www.codeship.com/projects/{{projectId}}/builds/{{id}}'>
-          <div class='branch_name'><strong>{{ branch }}</strong></div>
-        </a>
+        <div class='branch_name'>
+          <a target='codeship' href='http://www.codeship.com/projects/{{projectId}}/builds/{{id}}'>
+            <strong>{{ branch }}</strong>
+          </a>
+          <a target='commit' href='https://github.com/{{repositoryName}}/commit/{{commit_id}}'>
+            <strong>(github)</strong>
```

Comment:
Can we make this display the 7-character abbreviated SHA of the commit?

Looks like it should be just `{{commit_id}}`, right?